pkg/util/test/util.go

```diff
@@ -867,6 +868,15 @@ func ValidateStorageCluster(
 
 func validatePxPodsAnnotations(cluster *corev1.StorageCluster, timeout, interval time.Duration) error {
 	logrus.Debug("Validate PX pods..")
+	opVersion, err := GetPxOperatorVersion()
+	if err != nil {
+		logrus.Errorf("Operator version not found!")
+		return err
+	}
+	if opVersion.GreaterThanOrEqual(opVer23_10) {
```

Comment:
This is wrong statement, it needs to be lower then 23.10 and only then we skip it, 23.10.0 or higher is good and we should not skip it

Reply:
Ah. Updated.